RELEASE_NOTES.md

```diff
@@ -4,6 +4,11 @@ Release Notes
 v1.0.0-beta.x.x
 ---------------
 
+### Breaking changes
+
+- Added new required core API method `entityTraits`.
```

Comment:
[fix] Can we elaborate a little here - eg that its a manager method, and whats it's for.